src/_shared/scss/_adverts-jobs.scss

```diff
@@ -1,4 +1,10 @@
 .adverts--jobs {
+    @include mq($until: tablet) {
+        .adverts__row > :nth-child(2) ~ * {
```

Comment:
This is to match the same selector in `adverts.scss`, I didn't want to overthink it 😅